man/Burden-Estimate-Sets.Rd

```diff
@@ -0,0 +1,17 @@
+% Generated by roxygen2: do not edit by hand
```

Comment:
this looks suspicious - roxygen error?

Reply:
Error in the opening comments of burden.R caused this...